amethyst_network/src/error.rs

```diff
@@ -0,0 +1,85 @@
+//! Module containing error handling logic.
+
+use std::{
+    fmt::{self, Display, Formatter},
+    io,
+    sync::mpsc,
+};
+
+use crate::server::ServerSocketEvent;
+
+/// The `amethyst_network` result type.
+pub type Error<T> = Result<T, ErrorKind>;
```

Comment:
This is confusing; please call it `Result`.

Reply:
Consequently, `ErrorKind` can be called `Error`. `ErrorKind` is usually used for tags-only.

Reply:
Thanks for pointing out it should be indeed Result, must have accidentally renamed it to Error